ci-operator/config/openshift/csi-driver-shared-resource/openshift-csi-driver-shared-resource-master.yaml

```
@@ -15,21 +15,22 @@ binary_build_commands: make build
 build_root:
   from_repository: true
 images:
-- context_dir: images
-  dockerfile_path: Dockerfile
+- dockerfile_path: images/Dockerfile
   from: base
   inputs:
     ocp_builder_rhel-8-golang-1.17-openshift-4.11:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11
   to: csi-driver-shared-resource
-- context_dir: images
-  dockerfile_path: Dockerfile.mustgather
+- dockerfile_path: images/Dockerfile.mustgather
```

```suggestion
- dockerfile_path: Dockerfile.mustgather
```

@jkhelil you'll have to refactor https://github.com/openshift/csi-driver-shared-resource/pull/92 so that the dockerfiles live in the repo's top level dir

---

done with the correct path